admin/tests/unit/components/users/certification-center-memberships_test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import createGlimmerComponent from '../../../helpers/create-glimmer-component';
+
+module('Unit | Component | users | certification-center-memberships', function (hooks) {
+  setupTest(hooks);
+
+  test('#orderedCertificationCenterMemberships', function (assert) {
```

Comment:
**todo**
Ici ce serait plutôt un bloc `module`, avec à l'intérieur un bloc `test` et la description de ce que fait cette méthode ("it returns cc memberships in alphabetical order").

Reply:
Modification faite !